apps/angular/src/app/features/anime/table/table.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { Anime } from '@js-camp/core/models/anime';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { Observable } from 'rxjs';
+
+/** Table of anime. */
+@Component({
+	selector: 'camp-table',
+	templateUrl: './table.component.html',
+	styleUrls: ['./table.component.css'],
+})
+export class TableComponent {
+	/** Columns of table to display. */
+	protected readonly displayedColumns: readonly string[] = [
+		'image',
+		'titleEng',
+		'titleJpn',
+		'airedStart',
+		'type',
+		'status',
+	];
+
+	/** List of anime. */
+	protected readonly animeList$: Observable<Pagination<Anime>>;
+
+	public constructor(private animeService: AnimeService) {
```

Comment:
```suggestion
	public constructor(private readonly animeService: AnimeService) {
```

If you are not planning to use `animeService` outside of constructor, then you can omit `private` modifier.

```suggestion
	public constructor(animeService: AnimeService) {
```

____

Also, we can use `inject` from angular instead of injecting into constructor. This is recommended way to inject dependencies

```ts
export class TableComponent {
  private readonly animeService = inject(AnimeService);
  
  // other code ...
}
```